### PR TITLE
Added tabs to release modal and content for each one

### PR DESCRIPTION
**Pull request by prykon**
Upgraded the release notifications modal with tabbed browsing and different sections.

https://user-images.githubusercontent.com/36511666/184422594-70a1248d-95b9-4a89-9ff8-9be4a4fe4083.mov

### CODE REVIEW

**Comment by prykon** [comment]
Added icons to 'Translations' and 'Get Involved' tabs.

image

**Comment by corsacca** [comment]
Nice work on this @prykon !
A couple suggestions:
- [ ] lets cache the feed results like we do for the theme release.
- [ ] lets rename the 2nd tab "other news". Includes plugins + community news
- [ ] delete the translations tab
- [ ] remove icon from Get Involved tab
- [ ] Change content for get involved tab to look like

### [give icon] Give 
Make Disciple.Tools better by joining us financially. 

### [translate icon] Translate
Translate Disciple.Tools into you language or help make a translation better [here](https://poeditor.com/join/project/KcPvw3oaKD)

### [build icon] Build or Document
Make Disciple.Tools better by building a feature or helping us make the documentation better.
Find out more here [here](https://disciple.tools/join-the-community/)

**Comment by prykon** [comment]
@corsacca Something like this?

Screen Shot 2022-08-29 at 15 09 30

**Comment by corsacca** [comment]
looks great @prykon.

Can you run gulp styles ?

**Comment by corsacca** [comment]
@prykon 

Looks like you reverted my commit: https://github.com/DiscipleTools/disciple-tools-theme/pull/1765/commits/b41fc79ea1682b446230a6fa1fd38f8e6ef5d78a

Also the master needs to be pulled in and styles rebuilt `gulp styles`

**Comment by corsacca** [comment]
Nice work @prykon !

File: dt-core/release-notifications.php [hunk 1]
```diff
@@ -71,8 +124,35 @@ function dt_release_modal() {
 }
 add_action( 'wp_head', 'dt_release_modal' );
 
+function dt_get_plugins_news_links() {
+    $plugin_news_url = 'https://disciple.tools/news-categories/dt-plugin-releases/feed/';
```
Comment (corsacca) [inline dt-core/release-notifications.php:128]: @prykon is there a feed for all the news except the theme news?

Comment (prykon) [reply dt-core/release-notifications.php:128]: WordPress has feeds for categories and the generic one. I never heard of a feed sans a specific category, but I'll double check.

Comment (corsacca) [reply dt-core/release-notifications.php:128]: @prykon Or we create a type on the https://disciple.tools/newsb that is just "other"

**Comment by corsacca** [comment]
thank you @prykon!

**Pull request merged**